gitpod-io/website
# fix(lang/js): Invalid nodejs version

## Description
The nodejs version `14.08` seems to be nonexistent. Hence causing trouble to the users who are trying to follow this guide:
https://www.gitpod.io/docs/languages/javascript#node-versions

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/7478

## How to test

- Create a Gitpod workspace with a custom dockerfile:
```dockerfile
FROM gitpod/workspace-full:latest

RUN bash -c 'VERSION="14.8.0" \
    && source $HOME/.nvm/nvm.sh && nvm install $VERSION \
    && nvm use $VERSION && nvm alias default $VERSION'

RUN echo "nvm use default &>/dev/null" >> ~/.bashrc.d/51-nvm-fix
```
- Run `node --version`
- Check if it reports `14.8.0` as version.

## Release Notes

```release-note
Use a proper nodejs version in docs/lang/js as example
```

## Documentation



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1431"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
